File: Working with API help page controller action description in Web API/[C#]-Working with API help page controller action description in Web API/C#/ControllerActionDescriptions/Startup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.Owin;
using Owin;

[assembly: OwinStartup(typeof(ControllerActionDescriptions.Startup))]

namespace ControllerActionDescriptions
{
    public partial class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            ConfigureAuth(app);
        }
    }
}

File: SignalR con VB - Ejemplo Practico y Simple/[VB]-SignalR con VB - Ejemplo Practico y Simple/VB/SignaIR/Class/Startup.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Net
Imports System.Threading.Tasks
Imports System.Web
Imports System.Web.Routing
Imports Microsoft.AspNet.SignalR
Imports Microsoft.Owin.Security
Imports Owin
Imports Microsoft.Owin

<Assembly: OwinStartup(GetType(NotificacionesSignalr.Startup))> 
Namespace NotificacionesSignalr
    Public Class Startup
        Public Sub Configuration(app As IAppBuilder)
            ' Any connection or hub wire up and configuration should go here
            app.MapSignalR()
        End Sub
    End Class
End Namespace

